File: GridView/GridObjectRelationalCRUD/gridobjectrelationalcrud-cs/GridObjectRelationalCRUD-CS/Model/Album.cs
using System;
using System.Collections.Generic;

namespace GridObjectRelationalCRUD.Models
{
    [Serializable]
    public class Album 
    {
        private int id;
        private int artistId;
        private string title;
        private List<Track> tracks = new List<Track>();

        public Album()
        {

        }

        public Album(int id, int artistId, string title)
        {
            this.id = id;
            this.artistId = artistId;
            this.title = title;
        }

        public int Id
        {
            get { return id; }
            set { id = value; }
        }

        public int ArtistId
        {
            get { return artistId; }
            set { artistId = value; }
        }

        public string Title
        {
            get { return title; }
            set { title = value; }
        }

        public List<Track> Tracks
        {
            get { return this.tracks; }
        }
    }
}

File: GridView/GridObjectRelationalCRUD/gridobjectrelationalcrud-vb/GridObjectRelationalCRUD-VB/Model/Album.vb
<Serializable> _
Public Class Album
    Private m_id As Integer
    Private m_artistId As Integer
    Private m_title As String
    Private m_tracks As New List(Of Track)()


    Public Sub New()
    End Sub

    Public Sub New(id As Integer, artistId As Integer, title As String)
        Me.m_id = id
        Me.m_artistId = artistId
        Me.m_title = title
    End Sub

    Public Property Id() As Integer
        Get
            Return m_id
        End Get
        Set(value As Integer)
            m_id = value
        End Set
    End Property

    Public Property ArtistId() As Integer
        Get
            Return m_artistId
        End Get
        Set(value As Integer)
            m_artistId = value
        End Set
    End Property

    Public Property Title() As String
        Get
            Return m_title
        End Get
        Set(value As String)
            m_title = value
        End Set
    End Property

    Public ReadOnly Property Tracks() As List(Of Track)
        Get
            Return Me.m_tracks
        End Get
    End Property
End Class
